Lotto/Lotto/Lotto/LottoInputView.swift

```diff
@@ -18,6 +18,21 @@ struct LottoInputView {
         return read(with: "구매금액을 입력해 주세요.")
     }
     
+    static func readManualNumber() -> Int {
+        return Int(read(with: "수동으로 구매할 로또 수를 입력해 주세요.")) ?? 0
+    }
+    
+    static func readManualLotto(manualNumber: Int) -> [String] {
```

Comment:
```suggestion
    static func readManualLotto(number: Int) -> [String] {
```
manual 이 중복되어, number로만 표현해도 괜찮아보여요.😊